frontend/src/metabase/actions/components/ActionViz/Action.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useMemo, useCallback } from "react";
+import { t } from "ttag";
+import { connect } from "react-redux";
+
+import { executeRowAction } from "metabase/dashboard/actions";
+
+import type {
+  ActionDashboardCard,
+  ParametersForActionExecution,
+  WritebackQueryAction,
+  Dashboard,
+} from "metabase-types/api";
+import type { VisualizationProps } from "metabase-types/types/Visualization";
+import type { Dispatch } from "metabase-types/store";
+import type { ParameterValueOrArray } from "metabase-types/types/Parameter";
+
+import { generateFieldSettingsFromParameters } from "metabase/actions/containers/ActionCreator/FormCreator";
```

Comment:
Not sure if that's a big deal, but it feels a bit off that `actions/components` import something from `actions/containers`